packages/tools/client-debug-view/src/components/data-object-views/SharedCounterView.tsx

```diff
@@ -37,12 +54,65 @@ export function SharedCounterView(props: SharedCounterViewProps): React.ReactEle
 		};
 	}, [sharedCounter, setValue]);
 
+
+    function decrementCounter(): void {
+        sharedCounter.increment(-deltaValue);
+	}
+
+    function incrementCounter(): void {
+		sharedCounter.increment(deltaValue);
+	}
+
+    const inputSetValue = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        numberFormatCheck(e.target.value);
+    }
+
+    const numberFormatCheck = (val: string): void => {
```

Comment:
Nit: "numberFormatCheck" to me implies that the function just does a boolean check or some sort of validation. Since this function also updates (or doesn't update) our tracked delta value state, it might be worth naming this something else ("updateDeltaIfValid" or something), or just hoisting its logic up into `inputSetValue`.